_kbarticles/2022-12-08-common-kibana-searches-on-clouddotgov.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "Common Kibana Searches on cloud.gov"
```

Comment:
```suggestion
title: "Understanding Kibana and how to visualize your application logs"
```

Let's make the title really specific to the customer value of the article. And I think this article is really valuable